src/core/TimePicker/TimePicker.tsx

```diff
@@ -0,0 +1,233 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import '@itwin/itwinui-css/css/date-picker.css';
+
+const isSameHour = (hours: number, date: Date | undefined) => {
+  return !!date && hours === date.getHours();
+};
+
+const areSameMinutes = (minutes: number, date: Date | undefined) => {
+  return !!date && minutes === date.getMinutes();
+};
+
+const areSameSeconds = (seconds: number, date: Date | undefined) => {
+  return !!date && seconds === date.getSeconds();
+};
+
+const isSamePeriod = (period: string, date: Date | undefined) => {
```

Comment:
This is called "meridiem" btw

Reply:
I know the latin word in there, but can I just call them like that? I think I found this called period in some lib of many...